### PR TITLE
Add compat  data for HTMLSourceElement

### PR DESCRIPTION
**Pull request by gsouquet**
https://developer.mozilla.org/en-US/docs/Web/API/HTMLSourceElement

### CODE REVIEW

File: api/HTMLSourceElement.json [hunk 1]
```diff
@@ -0,0 +1,298 @@
+{
+  "api": {
+    "HTMLSourceElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLSourceElement",
+        "support": {
+          "chrome": {
+            "version_added": true
+          },
```
Comment (jpmedley) [inline api/HTMLSourceElement.json]: Please add a `chrome_android` with a value `true`.

File: api/HTMLSourceElement.json [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "keySystem": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLSourceElement/keySystem",
+          "support": {
+            "edge": {
```
Comment (jpmedley) [inline api/HTMLSourceElement.json]: Please add `webview_android`, `chrome`, and `chrome_android` with the value `false`.

File: api/HTMLSourceElement.json [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "sizes": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/sizes",
+          "support": {
+            "edge": {
```
Comment (jpmedley) [inline api/HTMLSourceElement.json]: Please add `webview_android`, `chrome`, and `chrome_android` with the value `true`.

File: api/HTMLSourceElement.json [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "srcset": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLSourceElement/srcset",
+          "support": {
+            "edge": {
```
Comment (jpmedley) [inline api/HTMLSourceElement.json]: Please add `webview_android`, `chrome`, and `chrome_android` with the value `true`.

**Comment by gsouquet** [comment]
Just fixed the requested changes, thanks @jpmedley

**Pull request merged**